public/service_portal/v0.1.0/openapi.json

```diff
@@ -967,6 +967,36 @@
                         "readOnly": true,
                         "title": "State",
                         "example": "The current state of a portfolio item"
+                    },
+                    "display_name": {
+                        "type": "string",
```

Comment:
@lindgrenj6 - Do we also know if editing these values is correctly handled in this doc as well?  E.g. the `POST /portfolio_items/{portfolio_item_id}` endpoint.

Reply:
Yep - right now we don't support editing the portfolio item objects (I assume since we pull data from topology mainly).

Reply:
We do not support it today from the UI, but the goal is to allow editing of the properties.  We pull the defaults from the service_offering and then an admin can modify what is presented to the user.

Reply:
Ok - currently it isn't supported in the API (i.e. we don't have PUT/PATCH support for `/portfolio_items/{portfolio_id}`, should we add a story for this in the backlog?

Reply:
Yes, we could use a story for this.

@syncrou We only need to support PATCH, correct?

Reply:
@gmcculloug - Correct.  We'll get a story added.